en/docs/samples/regular-webapp-saml-sample.md

```diff
@@ -0,0 +1,46 @@
+# Enable Authentication for SAML Web Application
+
+This page guides you through enabling authentication to a SAML web application using a **sample application** called Pickup. 
+
+----
+If you have have your own application, click the button below.
+
+<a class="samplebtn_a" href="../../authenticate/regular-webapp-oidc" target="_blank" rel="nofollow noopener">I have my own application</a>
+
+----
+
+{!samples/pickup-dispatch-saml.md!}
+
+----
+
+## Log in to the application
+
+1. Start the Tomcat server and access the following URL on your browser: <http://wso2is.local:8080/saml2-web-app-pickup-dispatch.com>.
+
+	```
+	http://<TOMCAT_HOST>:<TOMCAT_PORT>/saml2-web-app-pickup-dispatch.com
```

Comment:
Shall we use simple letters to represent the host and port to make the consistency?